gluten-core/src/main/scala/io/glutenproject/expression/ExpressionMappings.scala

```diff
@@ -39,8 +39,8 @@ object ExpressionMappings {
   final val BLOOM_FILTER_AGG = "bloom_filter_agg"
   final val VAR_SAMP = "var_samp"
   final val VAR_POP = "var_pop"
-  final val BIT_AND_AGG = "bitwise_and_agg"
-  final val BIT_OR_AGG = "bitwise_or_agg"
+  final val BIT_AND_AGG = "bit_and"
+  final val BIT_OR_AGG = "bit_or"
```

Comment:
@Yohahaha It seems that the Velox UT failed due to these changes. "Validation failed due to bit_and was not supported in AggregateRel."
Could we also align `BIT_AND_AGG` and `BIT_OR_AGG` to conform with Spark nodeName?

Reply:
Gluten + Velox reuse some Prestosql's functions currently, unfortunately bitwise_or_agg is that function.

I think Velox backend and CH backend should use different function mapping list in their own module, CC @zhouyuan @FelixYBW

Reply:
Anyway, let's open a issue for this.

Reply:
OK, I see. Is it possible to support it in the Velox backend by registering aliases, while still keeping unified function mapping list in gluten-core?